vault/logical_system_paths.go

```diff
@@ -1388,6 +1388,11 @@ func (b *SystemBackend) monitorPath() *framework.Path {
 				Description: "Log level to view system logs at. Currently supported values are \"trace\", \"debug\", \"info\", \"warn\", \"error\".",
 				Query:       true,
 			},
+			"log_format": {
+				Type:        framework.TypeString,
```

Comment:
I think you should add a default of `standard` in case an older vault cli is talking to a version that supports this.